pr_body.md

```
@@ -0,0 +1,10 @@
+﻿Implements present-LOD fallback (hysteresis + blend) with telemetry and CLI wiring.
+
+Highlights
+- C-ABI: policy/state in core/include/zx/zx_lod.h; impl in core/src/zx_lod.cpp
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)